aviary/utils/process_input_decks.py

```diff
@@ -167,56 +165,28 @@ def update_dependent_options(aircraft_values: AviaryValues(), dependent_options)
-    reserves = aircraft_values.get_val(
-        Aircraft.Design.RESERVES, units='lbm') if initial_guesses['reserves'] == 0 else initial_guesses['reserves']
-    if reserves_option == Reserves_Type.Set_Direct:
-        aircraft_values.set_val(Aircraft.Design.RESERVES, reserves, units='lbm')
-        if reserves < 10:
-            raise ValueError(
-                f'"{Aircraft.Design.RESERVES}" must be greater than 10 lbm.')
-    elif reserves_option == Reserves_Type.Set_Fraction:
-        if initial_guesses['reserves'] == 0:
+    if initial_guesses['reserves'] == 0:
+        reserves = aircraft_values.get_val(
+            Aircraft.Design.FIXED_RESERVES_FUEL, units='lbm')
+        reserves_option = "direct_val"
+        if reserves == 0:
             reserves = aircraft_values.get_val(
                 Aircraft.Design.RESERVES_FRACTION, units='unitless')
-        else:
-            reserves = initial_guesses['reserves']
-            aircraft_values.set_val(Aircraft.Design.RESERVES_FRACTION,
-                                    reserves, units='unitless')
-        if reserves < -1 or reserves > 0:
-            raise ValueError(
-                f'"{Aircraft.Design.RESERVES_FRACTION}" must be in [-1, 0].')
+            reserves_option = "fraction_val"
     else:
-        # if Aircraft.Design.RESERVES_FRACTION is not set,
-        # then determine reserves based on initial_guesses['reserves']
-        if initial_guesses['reserves'] > 10:
-            reserves = initial_guesses['reserves']
-            reserves_option = Reserves_Type.Set_Direct
-            aircraft_values.set_val(Aircraft.Design.RESERVES, reserves, units='lbm')
-        elif initial_guesses['reserves'] < 0:
-            reserves = initial_guesses['reserves']
-            reserves_option = Reserves_Type.Set_Fraction
-            aircraft_values.set_val(Aircraft.Design.RESERVES_FRACTION,
-                                    reserves, units='unitless')
-        elif initial_guesses['reserves'] == 0 and reserves > 10:
-            reserves_option = Reserves_Type.Set_Direct
-            aircraft_values.set_val(Aircraft.Design.RESERVES, reserves, units='lbm')
-        elif initial_guesses['reserves'] == 0 and reserves < 0:
-            reserves_option = Reserves_Type.Set_Fraction
-            aircraft_values.set_val(Aircraft.Design.RESERVES_FRACTION,
-                                    reserves, units='unitless')
-        aircraft_values.set_val(Aircraft.Design.RESERVES_OPTION,
-                                reserves_option, units='unitless')
+        reserves = initial_guesses['reserves']
+        if reserves < 0.0:
+            raise ValueError('initial_guesses["reserves"] must be greater than 0.')
+        elif reserves > 10:
+            reserves_option = "direct_val"
+        else:
+            reserves_option = "fraction_val"
 
     num_pax = aircraft_values.get_val(Aircraft.CrewPayload.NUM_PASSENGERS)
```

Comment:
```suggestion
```

Reply:
`num_pax` is used in the next blocks. I can not remove it.